### ISSUE TITLE
[HOLD for payment 2024-06-20] [$250] Quick action - Submit button displays different amount compared to amount in BNP

### ISSUE THREAD

**Issue by izarutskaya** [Expensify/App#42283]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.74-1
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Issue reported by:** Applause-Internal team

## Action Performed:
1. Go to staging.new.expensify.com
2. Submit an expense to a user.
3. Go to FAB.
4. Click Submit expense under Quick action.
5. Enter amount.
6. Select a different currency.
7. Change the amount.

## Expected Result:
The Submit button should just display Submit without the amount.

## Actual Result:
After changing to amount B, the submit button still shows amount A, which can be confusing.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/115492554/fdf8f28d-51b4-4283-8880-75312e820d6f

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0180b3b2cbd28bb97e</li>
        <li>Upwork Job ID: 1798112401827979264</li>
        <li>Last Price Increase: 2024-06-04</li>
<li>Automatic offers: </li>
<ul>
<li>dukenv0307 | Reviewer | 102636019</li>
<li>cretadn22 | Contributor | 102636020</li>
</ul></li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @kevinksullivan</details>

**Comment by kevinksullivan** [comment]
chatting about this one here

https://expensify.slack.com/archives/C036QM0SLJK/p1716247123666409

**Comment by kevinksullivan** [comment]
Updated the expected result, so we want to change this page so that it does not show an amount in the submit button. That way we don't have to "live update" the button as the amount changes.

**Comment by Beamanator** [comment]
Thanks for your proposals @cretadn22 and @dominictb 🙏 

I am going to quickly bring this to slack to get some other opinions how best to resolve this dispute, so please give me a day to come back with our updates 🙏

**Comment by Beamanator** [comment]
Alright I'm back with an update.

First, yes I can see the point that it would be nice to fix both bugs in this issue. However, @cretadn22's proposal was chosen and sounds good for fixing **this specific issue**, and it looks like we have another issue now for the 2nd bug (https://github.com/Expensify/App/issues/43215) where @dominictb has already posted their proposal.

So @dukenv0307 let's hire @cretadn22 only to fix the specific bug for this issue, and the proposal for the other bug will be handled in the other issue

**Comment by kevinksullivan** [comment]
@dukenv0307 can you complete the checklist above please?

**Comment by kevinksullivan** [comment]
Payments made to @cretadn22 and @dukenv0307 , just waiting on the checklist's completion.

**Comment by kevinksullivan** [comment]
Thanks @dukenv0307 ! All set here.
